samples/apps/copilot-chat-app/deploy/main.bicep

```diff
@@ -208,6 +208,10 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         name: 'ChatStore:Cosmos:ChatMemorySourcesContainer'
         value: 'chatmemorysources'
       }
+      {
```

Comment:
Please update the corresponding JSON ARM template too using the following command:
az bicep build -f .\main.bicep

Let me know if you want me to give you the JSON file instead.

The JSON file is needed by the "Deploy to Azure" button